docs/configuration.asciidoc

```diff
@@ -172,8 +175,6 @@ Click on a key to get more information.
 ** <<config-metric-set-limit>>
 ** <<config-agent-reporter-health-metrics>>
 ** <<config-agent-background-overhead-metrics>>
-* <<config-mongodb>>
```

Comment:
I was leaving it deliberately absent because we don't need to maintain every niche link, you think it's really necessary?

Reply:
We have to avoid broken links because the global Elastic docs build checks them, I thought it was enforced by the `elasticsearch-ci/docs` step but it seems it's not, however I can definitely remember broken links to be an extra issue when doing a release.

It gets even more tricky when you take in account the `1.x` branch as we can't update both `main` and `1.x` simultaneously without introducing broken links.